src/styles/blocks/card.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $main-card-width + 100px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding-top: 32px;
+
+  font-family: 'Roboto ', sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+
+  transition: $main-transition;
+
+  &__img {
+    width: $width-img;
+    height: $hight-img;
```

Comment:
There seems to be a typo in the variable name '$hight-img'. It should likely be '$height-img' to correctly represent the property it is modifying.